cluster-autoscaler/provisioningrequest/orchestrator/orchestrator_test.go

```diff
@@ -378,11 +378,11 @@ func TestScaleUp(t *testing.T) {
 			name:               "process atomic scale-up requests where batch processing of check capacity requests is enabled and check capacity requests are present in cluster",
 			provReqs:           []*provreqwrapper.ProvisioningRequest{newCheckCapacityMemProvReq, newCheckCapacityCpuProvReq, atomicScaleUpProvReq},
 			provReqToScaleUp:   atomicScaleUpProvReq,
-			scaleUpResult:      status.ScaleUpNotNeeded,
```

Comment:
Out of curiosity, why does this test case need to be changed?

Reply:
Previously, we would break [here](https://github.com/kubernetes/autoscaler/blob/16210e3da03def7e7cd5cc55012dd2ff6ffd8683/cluster-autoscaler/provisioningrequest/checkcapacity/provisioningclass.go#L109-L111), and not process any CheckCapacity, since the pods passed in were for atomic scale up. Now, since batch is enabled, don't use the passed in pods to determine the provisioning request(s), and successfully process some CheckCapacity requests